kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -68,6 +68,8 @@
   public static final String KUBERNETES_ALL_PROXY = "all.proxy";
   public static final String KUBERNETES_NO_PROXY = "no.proxy";
 
+  public static final String KUBERNETES_USER_AGENT = "fabric8-kubernetes-client";
```

Comment:
Wonder if we can compile in the version number somehow? That would help with debugging & issue tracking. Something like fabric8-kubernetes-client/version would be awesome. Wonder if @iocanel has any idea how we can do that?